README.md

```diff
@@ -119,6 +121,25 @@ This is an example of the value of `Specberus.meta` after the execution of `Spec
 }
 ```
 
+## REST API
+
+Similar to the [Node.js API](#js-api), Specberus exposes a REST API too.
+
+The endpoint is `<host>/api/`.
+
+There are three `GET` methods available:
+
+* `version`: returns the version string, eg `1.5.3`.
+* `metadata`: extract all known metadata from document; [returns JSON object with inferred properties](#extractmetadataoptions).
+* `validate`: check the document ([syntax](#validateoptions)); fails and returns an array of errors, or succeeds and returns the profile.
+
+Examples:
+
+* `<host>/api/version`
+* `<host>/api/metadata?url=http://example.com/doc.html`
+* `<host>/api/validate?file=/home/me/docs/spec.html`
```

Comment:
That caveat about `file` and `document` isn't new. Those two options don't work for users of our public instance of Specberus, but may come handy in other circumstances, eg we use `file` for testing the rules (and I'm using it now to test the REST API too). Or somebody may want to install Specberus locally and pass along `file`s.

@deniak, I'll modify my tests to test both with `file` and `url`, and maybe exclude the other two from the REST API, OK?